src/CMakeLists.txt

```diff
@@ -51,7 +51,7 @@ if (“${PROJECT_SOURCE_DIR}” STREQUAL “${CMAKE_SOURCE_DIR}”)
     endif()
 
     set(ENABLE_ASTYLE      OFF CACHE BOOL "")
-    set(ENABLE_CLANGFORMAT OFF CACHE BOOL "")
+    set(ENABLE_UNCRUSFITY  OFF CACHE BOOL "")
```

Comment:
```suggestion
    set(ENABLE_UNCRUSTIFY  OFF CACHE BOOL "")
```

Reply:
Good question @agcapps  -- `blt` will check for an `uncrustify` executable unless you explicitly diasble it with `ENABLE_UNCRUSTIFY=OFF`.  This line is explicitly disabling uncrustify since axom is no longer using it.